spacepy/coordinates.py

```diff
@@ -459,7 +459,7 @@ def from_skycoord(cls, skycoord):
             raise e.__class__("This method requires Astropy to be installed.")
 
         skycoord = astropy.coordinates.SkyCoord(skycoord).itrs  # Astropy ITRS is GEO
-        data = (skycoord.cartesian.xyz.to_value('m') / cls.Re).T  # convert Cartesian to Re units
+        data = (skycoord.cartesian.xyz.to('m').value / cls.Re).T  # convert Cartesian to Re units
```

Comment:
Thanks for that change.

The CI cancelled the tests on 3.5 because they're running on an Ubuntu 16.04 image - don't worry about that. The failures look like they need this same change made on lines 75, 94, 109 & 125 of `test_coordinates.py`, but I'll let you double-check that...

Reply:
Oops!  Changes included now